metaflow/plugins/aws/step_functions/step_functions.py

```diff
@@ -606,7 +607,11 @@ def _step_cli(self,
 
         script_name = os.path.basename(sys.argv[0])
         executable = self.environment.executable(node.name)
-        entrypoint = [executable, script_name]
+
+        if R.use_r():
```

Comment:
@jasonge27 Can we make this change a separate PR so that we can merge and make a Pythonic release for Metaflow?

Reply:
Yes here's the PR https://github.com/Netflix/metaflow/pull/317  @savingoyal